src/commands/images/buildImage.ts

```diff
@@ -20,37 +21,42 @@ export async function buildImage(context: IActionContext, dockerFileUri: vscode.
 
```

Comment:
Copying in Steven's comment since it didn't work right:

If the dockerFileUri !== undefined, then isn't the workspace folder implied by that uri?

Reply:
I'm not sure exactly. It might be possible (though horrible) to have two workspace folders open, one which is inside the other--in which case, both are equally valid as this Dockerfile's workspace folder.

I'll test it out.

Reply:
Turns out, yes you can. So in this case I'd prefer to just leave the old code untouched.

Reply:
Perhaps we should notify the user that they've made bad life choices if we encounter this nested workspace scenario.

Reply:
Probably, but this is a corner scenario already...quickPickWorkspaceFolder doesn't show a quick pick if there's only one choice, which is the overwhelming majority of cases.